### PR TITLE
test(demo-playwright): tests for addon-table

### PR DESCRIPTION
**Pull request by AtharvChandratre**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [x] Tests related changes
- [ ] Documentation content changes

## What is the current behaviour?

Addresses https://github.com/taiga-family/taiga-ui/issues/4912

## What is the new behaviour?
This PR adds an e2e test (with playwright) for Table

### CODE REVIEW

File: projects/demo-playwright/tests/addon-table/table/table.spec.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe(`Table`, () => {
+    test(`Dynamic column`, async ({page}) => {
+        await tuiGoto(page, `components/table`);
+        const example = new TuiDocumentationPagePO(page).getExample(`#dynamic`);
+        const dynamicButton = example.locator(`button`).first();
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/addon-table/table/table.spec.ts:8]: `addColumnButton` ? 

https://github.com/taiga-family/taiga-ui/pull/5664#discussion_r1365488125

File: projects/demo-playwright/tests/addon-table/table/table.spec.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+        await dynamicButton.click();
+        await expect(example).toHaveScreenshot(`01-table-dynamic-chromium-darwin.png`);
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/addon-table/table/table.spec.ts:11]:
```suggestion
        await expect(example).toHaveScreenshot(`01-table-dynamic.png`);
```

Comment (AtharvChandratre) [reply projects/demo-playwright/tests/addon-table/table/table.spec.ts]: Addressed

File: projects/demo-playwright/tests/addon-table/table/table.spec.ts (hunk 1 above)
Comment (AtharvChandratre) [reply projects/demo-playwright/tests/addon-table/table/table.spec.ts]: Addressed

**Comment by nsbarsukov** [comment]
Thanks for your contribution ❤️

**Review by splincode — APPROVED** [review]

**Pull request merged**